### DISCUSSION TITLE
pony trot

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#9] [Show and tell]
overall, i thouroghly enjoyed toonsquid w my first animation made in it ! ^_^ the only confusing part was the fill tool functions loll but i worked around it decently ! (if wondering of said confusion on fill tool; it was bc i had no idea if there was fill behind / referencing layers to color on a separate layer) but the workaround was just duplicating lineart layers lol

heres the [vod](https://www.twitch.tv/videos/1609305345?t=455s) of me streaming the first impressions of the app if ur curious ! i’ll definitely try doing an app review sometime soon ! ^^
![8B5309B4-F60D-4335-8757-D7510F6B2B14](https://user-images.githubusercontent.com/115050959/194184179-fba715a0-7cc8-4b57-a1b2-7f9d74bd97a0.gif)

**Comment by keiwando** [comment]
I love it! Thanks for sharing! I can't wait to watch the VOD when I find some time :)
Yeah, there is currently not an option to fill a layer based on line work on a different layer but you're not the first one to ask so it's slowly making its way up the priority list.
